schema/materials_category_components/operations/core/modifications/perturb.json

```diff
@@ -0,0 +1,16 @@
+{
+    "$id": "materials-category-components/operations/core/modifications/perturb",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Perturbation Schema",
+    "description": "A perturbation operation that modifies a configuration by applying a perturbation to atoms coordinates.",
+    "type": "object",
+    "properties": {
+        "material": {
+            "$ref": "../../../../material.json"
+        },
+        "perturbation_function": {
```

Comment:
It may make sense to define the function somewhere in a reusable part as a `function_3d` or something like that